.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+multi_line_output = 3
+include_trailing_comma = True
+force_grid_wrap = 0
+use_parentheses = True
+ensure_newline_before_comments = True
+line_length = 88
```

Comment:
It's a good practice to end with a newline character

Reply:
added newline char